docs/compatibility.md

```diff
@@ -37,6 +37,26 @@ HTTPX uses `utf-8` for encoding `str` request bodies. For example, when using `c
 
 For response bodies, assuming the server didn't send an explicit encoding then HTTPX will do its best to figure out an appropriate encoding. Unlike Requests which uses the `chardet` library, HTTPX relies on a plainer fallback strategy (basically attempting UTF-8, or using Windows-1252 as a fallback). This strategy should be robust enough to handle the vast majority of use cases.
 
+## Cookies
+
+If using a client instance, then cookies should always be set on the client rather than on a per-request basis.
+
+This usage is supported...
+
+```python
+client = httpx.Client(cookies=...)
+client.post(...)
+```
+
+This usage is deprecated...
```

Comment:
Colons rather than ellipses here perhaps?
```suggestion
This usage is supported:

```python
client = httpx.Client(cookies=...)
client.post(...)
```

This usage is deprecated:
```

Reply:
Ellipses are Tom's style. :-)

Reply:
Good enough for me, then!

Reply:
Oh, I don't know, I think it's probably a good suggestion. 😃